osu.Game/Skinning/LegacySkinConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Skinning
+{
+    public class LegacySkinConfiguration : DefaultSkinConfiguration
+    {
+        public const decimal LATEST_VERSION = 2.5m;
+
+        /// <summary>
+        /// Legacy version of this skin.
+        /// </summary>
+        public decimal? LegacyVersion { get; internal set; }
+    }
+
+    public enum LegacySkinConfigurations
```

Comment:
I'd sooner rename the `LegacySkinConfiguration` class than do this weirdness in naming.

Reply:
`StableSkinConfiguration`..?

Reply:
no, synonyms to avoid naming clashes are invalid

Reply:
how about renaming the whole configuration enums to `Property`; 
 - `LegacySkinConfigurations` -> `LegacySkinProperty`
 - `GlobalSkinConfiguration` -> `GlobalSkinProperty` | `SkinProperty`